app/helpers/openssl_helper.rb

```diff
@@ -0,0 +1,99 @@
+module OpensslHelper
+  def random_key_iv(size: 128, mode: :GCM)
+    cipher = OpenSSL::Cipher::AES.new(size, mode)
+    cipher.encrypt
+    {
+      key: cipher.random_key,
+      iv: cipher.random_iv,
+    }
+  end
```

Comment:
1. I think this name is a bit misleading.... this seems to be AES-GCM helper stuff instead of all of OpenSSL, we have our AesEncryptor class -- it depends on having a piece of `cek` random data already passed in, but it might help to try to combine these? No need to have all these AES defaults and lengths written twice.

2. I thought we were going to generate the AES keys client-side, and pass them to the server when the server needs them?